packages/core/src/routes/experience/classes/validators/provision-library.ts

```diff
@@ -38,11 +40,77 @@ export class ProvisionLibrary {
     private readonly ctx: WithLogContext
   ) {}
 
+  /**
+   * Insert a new user into the Logto database using the provided profile.
+   *
+   * - provision the organization for the new user based on the profile
```

Comment:
sounds not consecutive. also, i suspect "provision" has no meaning of creation, the function name should be revisited.